cookbook/import_export/create-connector.rst

```diff
@@ -43,40 +31,80 @@ Register the bundle in AppKernel:
         // ...
     }
 
-Configure our Job
------------------
+Create our Job
+--------------
+
+Create a file ``Resources/config/batch_jobs.yml`` in our Bundle to configure a new job:
+
+.. code-block:: yaml
+
+    connector:
+        name: Demo Connector
+        jobs:
+            demo_job:
+                title: acme_connector.jobs.demo_job.title
+                type:  export
+                steps:
+                    stepOne:
+                        title:         acme_connector.jobs.demo_job.export.title
+                        services:
+                            reader:    pim_connector.reader.dummy_item
+                            processor: pim_connector.processor.dummy_item
+                            writer:    pim_connector.writer.dummy_item
 
-Configure a job in ``Resources/config/batch_jobs.yml``:
+Here we use some existing dummy reader, processor and writer (they implement relevant interfaces and are useable but they do nothing with data).
```

Comment:
It would be more appropriate to say 'an existing dummy reader, a processor and a writer' , 'some' implies plural (readers, processors and writers).